src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -1728,6 +1728,40 @@ public void sendLeaveMessage(Player player, String quitMessage) {
         }
     }
 
+    /**
+     * Gives an "online" role to a player after joining. The role id can be added in the config.
+     *
+     * @param player the player
+     * @see #removePlayerOnlineRole(Player, String)
+     */
+    public void givePlayerOnlineRole(Player player, String RoleId) {
```

Comment:
Why does this need it's own role? This feels extremely redundant

Reply:
moved the retrieving role id functionality to give/removePlayerOnlineRole, as well as implemented some error catching to alert the user in case of an issue.